keycloak/themes/sunbird/login/keycloak/docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+KEYCLOAK_USER=admin
+KEYCLOAK_PASSWORD=ash12345678
```

Comment:
Keycloak credentials should not be part of this script or this commit. It is a security hole.